portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase

```diff
@@ -120,14 +120,34 @@ definition {
 	}
 
 	@description = "Veriy that the Field Label can be hidden"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test LabelCanBeHidden {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133040 LabelCanBeHidden pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Image");
+
+		Form.gotoAdvancedTab();
+
+		FormViewBuilder.clickShowLabel();
+
```

Comment:
Hi @chaowang-123,

according to the [test description](https://issues.liferay.com/browse/LPS-133040), the third and fifth steps are missing, which are:

> 3. Assert the "Image" label is present.

> 5. Assert the "Image" label is not present.

Please, use the `FormFields.viewFieldLabel` and `FormFields.viewFieldLabelNotPresent` macros to the cases where applicable.

In addition, you could replace `FormViewBuilder.clickShowLabel()` by `FormFields.disableSwitcher(fieldName = "Show Label")`, to keep only using macros that belongs to Forms product.

Thank you =)

Reply:
hi, @alinedoleron  have updateed, thanks